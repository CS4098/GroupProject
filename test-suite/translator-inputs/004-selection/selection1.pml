process test {
    selection {
        action act1 {
	        requires { a }
	        provides { a }
        }
        sequence one {
		    action act2 {
		        requires { a }
                provides { c }
		    }
            action act3 {
		        requires { c }
                provides { b }
		    }
	    }
    }
}

