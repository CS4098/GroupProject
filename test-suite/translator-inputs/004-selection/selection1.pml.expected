bool a;
bool b;
bool c;

active proctype test()
{
	if
	:: true ->
			a ->
				a = true;
	:: true ->
				a ->
					c = true;
				c ->
					b = true;
	fi
}
